apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataAdapter.cs

```diff
@@ -238,21 +259,33 @@ private SpannerCommand GetBuiltDeleteCommand()
             }
             if (_parsedParameterCollection != null)
             {
-                _builtDeleteCommand = SpannerConnection.CreateDeleteCommand(UpdateTable, _parsedParameterCollection);
+                _builtDeleteCommand =
+                    SpannerConnection.CreateDeleteCommand(AutoGeneratedCommandTable, _parsedParameterCollection);
             }
             return _builtDeleteCommand;
         }
 
+        private SpannerCommand GetBuiltSelectCommand()
+        {
+            if (_builtSelectCommand != null)
+            {
+                return _builtSelectCommand;
+            }
+            _builtSelectCommand = SpannerConnection.CreateSelectCommand($"SELECT * FROM {AutoGeneratedCommandTable}");
+            return _builtSelectCommand;
```

Comment:
Nit: for something like this, I usually like to have a single return at the end, but it's up to you:
```csharp
if (_builtSelectCommand == null)
{
    _builtSelectCommand = SpannerConnection.CreateSelectCommand($"SELECT * FROM {AutoGeneratedCommandTable}");
}
return _builtSelectCommand;
```